config/config.py

```diff
@@ -57,17 +57,17 @@ def _create_new_config_file():
     global _user_config_file
     if _user_config_file is not None:
         logging.error("Trying to create new config file when config file as already been set. Exiting")
-        exit(1)
+        return
 
```

Comment:
These exits that got changed to returns fix a weird behavior on windows where the gui would close immediately after launch if windows could not find the config file. Originally these were exits to stop the CLI uploader from uploading without a config file, but that error gets caught elsewhere and handled anyways so this change benefits both CLI and GUI.